From aa0a14db011c59116beb51413ee9a0b253e04f1a Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Mon, 10 Feb 2014 15:07:36 +0000
Subject: [PATCH 17/72] Fix gluster volume name extraction

The get_volume_names() function in the gluster plugin tries to
extract volume names from the output of the "gluster volume info"
command using a slice range. The range start should be 12 to
account for the "Volume Name:" label at the start of the line.

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/plugins/gluster.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sos/plugins/gluster.py b/sos/plugins/gluster.py
index 02dbac3..4286b59 100644
--- a/sos/plugins/gluster.py
+++ b/sos/plugins/gluster.py
@@ -39,7 +39,7 @@ class Gluster(Plugin, RedHatPlugin):
         for line in fp.readlines():
             if not line.startswith("Volume Name:"):
                 continue
-            volname = line[14:-1]
+            volname = line[12:-1]
             out.append(volname)
         fp.close()
         return out
-- 
1.9.3

